README.md

```diff
@@ -49,7 +49,19 @@ The media type for JSON Siren is `application/vnd.siren+json`.
       "fields": [
         { "name": "orderNumber", "type": "hidden", "value": "42" },
         { "name": "productCode", "type": "text" },
-        { "name": "quantity", "type": "number" }
+        { "name": "quantity", "type": "number" },
+        { "name": "color", "type": "radio", "title": "Choose a Color", 
```

Comment:
Thanks for spotting this. The 'color' attribute was not meant to appear twice. Changed to 'accessories'.